source/manuals/programming-languages/python/style-guide.html.md.erb

```diff
@@ -0,0 +1,45 @@
+---
+title: GDS Python Style Guide
+expires: living
+---
+
+# <%= current_page.data.title %>
+
+
+### Some rule
```

Comment:
Guessing you may not be intending to include this?

Reply:
Oh I see you did intend to include it as an example!

Reply:
I can smarten this up to make it clearer it's an example or remove it totally, your call. Really it was more of  a placeholder for me before I wrote the line length entry.

Reply:
I'm gonna remove it. The pattern should be clear from the 120 entry

Reply:
Done 👍